### ISSUE TITLE
Problem with JOSM-Template

### ISSUE THREAD

**Issue by lomonossof** [MarcusWolschon/osmeditor4android#531]
Hallo!

I'm using Vespucci with a Fire Phone with LineageOS (KitKat 4.4.4). I wanted to use this
http://www.openrailwaymap.org/josm-presets/de-signals-eso.zip
JOSM-preset but I just get an error message, and a warning that it won't work. Is there a chance to look into it?

Regards,

  lomonossof

**Comment by simonpoole** [comment]
The is a combination of a couple of issues, that I've fixed see https://github.com/MarcusWolschon/osmeditor4android/commit/07ee33d41dea8e0bce5c47cccaa7dbc28cefee84 mainly that the actual preset file in the archive is not called "preset.xml". 

Which Vespucci version are you using?

**Comment by lomonossof** [comment]
The filename was a good hint! I tried the following: Download the preset, open the zip-file and rename the xml to preset.xml. Then upload the archive to my own webspace and open this one from Vespucci and - tadaaa! It works.
Therefore I would suggest making a small change to Vespucci: It would be nice if it accepted presets where the xml-file has the same name as the archive (instead of preset.xml), as well. I did a quick look and found several presets with files named this way.

Regards!

    Thomas

(I'm using 0.9.8r1242 at this moment)
